git_diff.py

```diff
@@ -0,0 +1,63 @@
+import subprocess
+import json
+from git import Repo
+
+repo = Repo("./")
+
+
+def get_staged_files():
+    try:
+        # Run the git diff command to get staged files
+        result = subprocess.run(['git', 'diff', '--name-only'],
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE,
+                                text=True,
+                                check=True)
+        # Return the output, which is a list of file names
+        return result.stdout.splitlines()
+    except subprocess.CalledProcessError as e:
+        return [f"Error: {e.stderr.strip()}"]
+    except FileNotFoundError:
+        return ["Error: Git is not installed or not found in the PATH."]
+
+
+def is_version_only_change(file_path, repo):
```

Comment:
this `repo` argument is not used in the function

Reply:
I use it in line 30. It's essentially the same function used in the HBCB code.

Reply:
sounds great!